src/test/getarg_tests.cpp

```diff
@@ -30,6 +30,50 @@ static void ResetArgs(const std::string& strArg)
     gArgs.ParseParameters(vecChar.size(), vecChar.data());
 }
 
+BOOST_AUTO_TEST_CASE(configfile)
+{
+  fs::path expectedConfigPath(fs::system_complete("test/data/bitcoin.conf"));
```

Comment:
The practice for the unit tests has been to embed all test files in the source code - by including the generated test files (in this case `bitcoin.conf.raw.h`, I think).
The reason is that it should be possible to copy `test_bitcoin` to a new platform (for example by installing the distributed binaries) and run it without copying the entire source tree.
If you really need a file (for testing) instead of data in memory, create the file in a temporary directory.